app/controllers/api/v2/ext_app/containers_controller.rb

```diff
@@ -55,11 +55,7 @@ def container_params
           :fingerprint,
           :alias
         ]},
-        {bootstrappers: [
-          :bootstrap_type,
-          :bootstrap_cookbooks_url,
-          :bootstrap_attributes
-        ]}
+        :bootstrappers
```

Comment:
why don't we make this explicit?

Reply:
Hi mas, We readjust this change to https://github.com/pathfinder-cm/pathfinder-mono/pull/30/commits/37d014568fb4e923f66b72e67a77ba0263904b3e. 

Can you please check.